### PR TITLE
{bio}[system/system] IGV v2.7.0

### PR DESCRIPTION
**Pull request by pescobar**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/i/IGV/IGV-2.7.0-Java-11.eb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+# Modified by Adam Huffman
+# The Francis Crick Institute
+
+easyblock = 'Tarball'
+
+name = 'IGV'
+version = '2.7.0'
+versionsuffix = '-Java-%(javaver)s'
+
+homepage = 'http://www.broadinstitute.org/software/igv/'
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/i/IGV/IGV-2.7.0-Java-11.eb]:
```suggestion
homepage = 'https://www.broadinstitute.org/software/igv/'
```

File: easybuild/easyconfigs/i/IGV/IGV-2.7.0-Java-11.eb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+description = """This package contains command line utilities for
+ preprocessing, computing feature count density (coverage),  sorting, and
+ indexing data files."""
+
+toolchain = SYSTEM
+
+source_urls = ['http://data.broadinstitute.org/igv/projects/downloads/%(version_major)s.%(version_minor)s']
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/i/IGV/IGV-2.7.0-Java-11.eb]:
```suggestion
source_urls = ['https://data.broadinstitute.org/igv/projects/downloads/%(version_major)s.%(version_minor)s']
```

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/69fd129bdd2edbb8e741f490764a48f6 for a full test report.

**Comment by boegel** [comment]
closing this, superseded by merged PRs for more recent versions of IGV (see #9987, #12734, #15341)

**Pull request closed without merging**